sphinxcontrib/katex.py

```diff
@@ -70,16 +70,25 @@ def latex_defs_to_katex_macros(defs):
 
 def html_visit_math(self, node):
     self.body.append(self.starttag(node, 'span', '', CLASS='math'))
+    if 'latex' in node.attributes:
+        latex = node['latex']  # for Sphinx < 1.8.0
+    else:
+        latex = node.astext()  # for Sphinx >= 1.8.0
     self.body.append(self.builder.config.katex_inline[0] +
-                     self.encode(node['latex']) +
+                     self.encode(latex) +
                      self.builder.config.katex_inline[1] + '</span>')
     raise nodes.SkipNode
 
 
 def html_visit_displaymath(self, node):
     self.body.append(self.starttag(node, 'div', CLASS='math'))
+    if 'latex' in node.attributes:
+        latex = node['latex']  # for Sphinx < 1.8.0
+    else:
+        latex = node.astext()  # for Sphinx >= 1.8.0
+
```

Comment:
This replicates the code from above, maybe it would be better to replace by a function like:
```python
latex = latex_from_node(node)
```

Reply:
Sure, I modified the patch to this effect.